repos/fdbt-site/src/pages/selectCappedProductGroup.tsx

```diff
@@ -91,28 +92,21 @@ const SelectPassengerType = ({
     </TwoThirdsLayout>
 );
 
-export const getServerSideProps = (ctx: NextPageContextWithSession): { props: SelectCappedProductGroupProps } => {
+export const getServerSideProps = async (
+    ctx: NextPageContextWithSession,
```

Comment:
The Create new button links to the incorrect page (href="/viewGroupsOfProducts">)